bigquery/tests/unit/test_client.py

```diff
@@ -1753,6 +1753,23 @@ def test_list_jobs_w_project(self):
                 'projection': 'full',
             })
 
+    def test_list_jobs_w_time_filter(self):
+        creds = _make_credentials()
+        client = self._make_one(self.PROJECT, creds)
+        conn = client._connection = _make_connection({})
+
+        list(client.list_jobs(
+            min_creation_time=1, max_creation_time=1527874895820))
```

Comment:
I've got a preference for naming the majyk constants here.  It isn't a blocker for merge, but it does ease maintenance down the road.

Reply:
The value of the inputs is not important here because the function output is not being saved or tested in this test. This is just testing that the parameters exist and get passed to the API properly.

Reply:
I can agree with Tres, specifically for the max_creation since it does look magical. Does that number have any significance?

Reply:
I added some constants in the follow-up change https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5431